### ISSUE TITLE
TopMost option from Show-InstallationWelcome not working 

### ISSUE THREAD

**Issue by dersonc** [PSAppDeployToolkit/PSAppDeployToolkit#35]
I have the installation script set with the following parameters for Show-InstallationWelcome:
```
Show-InstallationWelcome -CloseApps 'iexplore,winword,notepad' -PromptToSave -AllowDeferCloseApps -ForceCloseAppsCountdown 3600 -BlockExecution -DeferTimes 2 -DeferDays 7 -PersistPrompt -TopMost 0
```

I also tried setting -TopMost to $false but I got the same results.

But when I run the installation the InstallationWelcome window is always forced to be on top of the other windows even if there are deferrals left.

This is the log from the execution:
```
[Initialization] :: *******************************************************************************	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: *******************************************************************************	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: [.NET.Framework.4.5.2.Multi-Targeting.Pack_4.5.51651_001] setup started.	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: Script [C:\Users\yyyyyyyy.zzzz\Desktop\AppDeployToolkit\AppDeployToolkitMain.ps1] dot-source invoked by [C:\Users\yyyyyyyy.zzzz\Desktop\Deploy-Application.ps1]	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: Script [C:\Users\yyyyyyyy.zzzz\Desktop\AppDeployToolkit\AppDeployToolkitExtensions.ps1] dot-source invoked by [C:\Users\yyyyyyyy.zzzz\Desktop\AppDeployToolkit\AppDeployToolkitMain.ps1]	PSAppDeployToolkitExt	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: [.NET.Framework.4.5.2.Multi-Targeting.Pack_4.5.51651_001] script version is [1.0.0]	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: [Deploy Application] script version is [3.6.1]	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: [App Deploy Toolkit Main] script version is [3.6.5]	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: [App Deploy Toolkit Extensions] version is [1.5.0]	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: Computer Name is [nnnnnnnnnn.zzzz.xxxxxxxxxxxxx.com]	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: Current User is [zzzz\yyyyyyyy]	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: OS Version is [Microsoft Windows 7 Enterprise Service Pack 1 64-bit 6.1.7601.65536]	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: OS Type is [Workstation]	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: Current Culture is [en-US] and UI language is [EN]	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: Hardware Platform is [Virtual:VMWare]	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: PowerShell Host is [ConsoleHost] with version [2.0]	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: PowerShell Version is [2.0 x64]	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: PowerShell CLR (.NET) version is [2.0.50727.5485]	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: *******************************************************************************	PSAppDeployToolkit	5/20/2015 3:58:00 PM	7976 (0x1F28)
[Initialization] :: Display session information for all logged on users: 


NTAccount           : zzzz\yyyyyyyy
SID                 : S-1-5-21-1454471165-2077806209-1801674531-7103031
UserName            : yyyyyyyy
DomainName          : zzzz
SessionId           : 1
SessionName         : Console
ConnectState        : Active
IsCurrentSession    : True
IsConsoleSession    : True
IsActiveUserSession : True
IsUserSession       : True
IsRdpSession        : False
IsLocalAdmin        : False
LogonTime           : 3/23/2015 1:19:06 PM
IdleTime            : 00:00:00
DisconnectTime      : 
ClientName          : 
ClientProtocolType  : 
ClientDirectory     : 
ClientBuildNumber   : 0



	PSAppDeployToolkit	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Initialization] :: The following users are logged on to the system: [zzzz\yyyyyyyy].	PSAppDeployToolkit	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Initialization] :: Current process is running with user account [zzzz\yyyyyyyy] under logged in user session for [zzzz\yyyyyyyy].	PSAppDeployToolkit	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Initialization] :: The following user is the console user [zzzz\yyyyyyyy] (user with control of physical monitor, keyboard, and mouse).	PSAppDeployToolkit	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Initialization] :: The active logged on user is [zzzz\yyyyyyyy].	PSAppDeployToolkit	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Initialization] :: The active logged on user [zzzz\yyyyyyyy] has a primary UI language of [EN].	PSAppDeployToolkit	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Initialization] :: The following UI messages were imported from the config XML file: [UI_Messages_EN].	PSAppDeployToolkit	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Initialization] :: The active logged on user [zzzz\yyyyyyyy] has a DPI scale factor of [100] with DPI pixels [96].	PSAppDeployToolkit	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Initialization] :: Unable to load COM Object [Microsoft.SMS.TSEnvironment]. Therefore, script is not currently running from a SCCM Task Sequence.	PSAppDeployToolkit	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Initialization] :: Skipping attempt to check for and make the task scheduler services healthy because the App Deployment Toolkit is not running under the [NT AUTHORITY\SYSTEM] account.	PSAppDeployToolkit	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Initialization] :: Session 0 not detected.	PSAppDeployToolkit	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Initialization] :: Installation is running in [Interactive] mode.	PSAppDeployToolkit	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Initialization] :: Deployment type is [Installation].	PSAppDeployToolkit	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: Get deferral history...	Get-DeferHistory	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: Return fully qualified registry key path [Registry::HKEY_LOCAL_MACHINE\SOFTWARE\xxxx xx xxxxxxx\Packages\PSAppDeployToolkit\DeferHistory\.NET.Framework.4.5.2.Multi-Targeting.Pack_4.5.51651_001].	Convert-RegistryPath	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: Get registry key [Registry::HKEY_LOCAL_MACHINE\SOFTWARE\xxxx xx xxxxxxx\Packages\PSAppDeployToolkit\DeferHistory\.NET.Framework.4.5.2.Multi-Targeting.Pack_4.5.51651_001] and all property values.	Get-RegistryKey	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: Defer history shows [2] deferrals remaining.	Show-InstallationWelcome	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: User has [1] deferrals remaining.	Show-InstallationWelcome	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: Defer history shows a deadline date of [2015-05-27 15:50:22Z].	Show-InstallationWelcome	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: Convert the date [05/27/2015 15:50:22] to a universal sortable date time pattern based on the current culture [en-US].	Get-UniversalDate	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: User has until [2015-05-27 15:50:22Z] before deferral expires.	Show-InstallationWelcome	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: Convert the date [05/20/2015 15:58:02] to a universal sortable date time pattern based on the current culture [en-US].	Get-UniversalDate	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: Check for running application(s) [iexplore,winword,notepad]...	Get-RunningProcesses	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: The following processes are running: [iexplore,notepad].	Get-RunningProcesses	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: Resolve process descriptions...	Get-RunningProcesses	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: Finished checking running application(s).	Get-RunningProcesses	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: Prompt user to close application(s) [Internet Explorer,Notepad]...	Show-WelcomePrompt	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: User has the option to defer.	Show-WelcomePrompt	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: Close applications countdown has [3600] seconds remaining.	Show-WelcomePrompt	5/20/2015 3:58:02 PM	7976 (0x1F28)
[Pre-Installation] :: Check for running application(s) [iexplore,winword,notepad]...	Get-RunningProcesses	5/20/2015 3:58:14 PM	7976 (0x1F28)
[Pre-Installation] :: The following processes are running: [iexplore,notepad].	Get-RunningProcesses	5/20/2015 3:58:14 PM	7976 (0x1F28)
[Pre-Installation] :: Resolve process descriptions...	Get-RunningProcesses	5/20/2015 3:58:14 PM	7976 (0x1F28)
[Pre-Installation] :: Finished checking running application(s).	Get-RunningProcesses	5/20/2015 3:58:14 PM	7976 (0x1F28)
[Pre-Installation] :: Prompt user to close application(s) [Internet Explorer,Notepad]...	Show-WelcomePrompt	5/20/2015 3:58:14 PM	7976 (0x1F28)
[Pre-Installation] :: User has the option to defer.	Show-WelcomePrompt	5/20/2015 3:58:14 PM	7976 (0x1F28)
[Pre-Installation] :: Close applications countdown has [3588] seconds remaining.	Show-WelcomePrompt	5/20/2015 3:58:14 PM	7976 (0x1F28)
[Pre-Installation] :: Installation deferred by the user.	Show-InstallationWelcome	5/20/2015 3:58:18 PM	7976 (0x1F28)
[Pre-Installation] :: Set deferral history: [DeferTimesRemaining = 1].	Set-DeferHistory	5/20/2015 3:58:18 PM	7976 (0x1F28)
[Pre-Installation] :: Return fully qualified registry key path [Registry::HKEY_LOCAL_MACHINE\SOFTWARE\xxxx xx xxxxxxx\Packages\PSAppDeployToolkit\DeferHistory\.NET.Framework.4.5.2.Multi-Targeting.Pack_4.5.51651_001].	Convert-RegistryPath	5/20/2015 3:58:18 PM	7976 (0x1F28)
[Pre-Installation] :: Update registry key value: [Registry::HKEY_LOCAL_MACHINE\SOFTWARE\xxxx xx xxxxxxx\Packages\PSAppDeployToolkit\DeferHistory\.NET.Framework.4.5.2.Multi-Targeting.Pack_4.5.51651_001] [DeferTimesRemaining = 1].	Set-RegistryKey	5/20/2015 3:58:18 PM	7976 (0x1F28)
[Pre-Installation] :: Set deferral history: [DeferDeadline = 2015-05-27 15:50:22Z].	Set-DeferHistory	5/20/2015 3:58:18 PM	7976 (0x1F28)
[Pre-Installation] :: Return fully qualified registry key path [Registry::HKEY_LOCAL_MACHINE\SOFTWARE\xxxx xx xxxxxxx\Packages\PSAppDeployToolkit\DeferHistory\.NET.Framework.4.5.2.Multi-Targeting.Pack_4.5.51651_001].	Convert-RegistryPath	5/20/2015 3:58:18 PM	7976 (0x1F28)
[Pre-Installation] :: Update registry key value: [Registry::HKEY_LOCAL_MACHINE\SOFTWARE\xxxx xx xxxxxxx\Packages\PSAppDeployToolkit\DeferHistory\.NET.Framework.4.5.2.Multi-Targeting.Pack_4.5.51651_001] [DeferDeadline = 2015-05-27 15:50:22Z].	Set-RegistryKey	5/20/2015 3:58:18 PM	7976 (0x1F28)
[Pre-Installation] :: .NET.Framework.4.5.2.Multi-Targeting.Pack_4.5.51651_001 Installation completed with exit code [65000].	Exit-Script	5/20/2015 3:58:18 PM	7976 (0x1F28)
[Pre-Installation] :: -------------------------------------------------------------------------------	Exit-Script	5/20/2015 3:58:18 PM	7976 (0x1F28)
```

**Comment by mmashwani** [comment]
This issue has been resolved in the latest 3.6.5 beta. Please test and confirm it is working as expected.

**Comment by dersonc** [comment]
It is working now, thank you! :smile:
